proto/cmp/types/v1alpha/search.proto

```diff
@@ -81,14 +81,18 @@ message SearchResponseMetadata {
   // to RFC 4122
   UUID search_id = 1;
 
-  // Context for Inventory system communication.
+  // Context for Inventory system concepts that need to be included in a search response,
+  // like an OwnerCode, PTC_OfferParameters, Tax codes, Disclosure RefID, etc. or a
+  // serialized combination of these codes.
   string context = 2;
 
   // Errors encountered in the generation of the response. Standardized error list
   // to be created.
   string errors = 3;
```

Comment:
If it's multiple errors, shouldn't it be a repeated field?

Reply:
adopted